packages/stringsifter/PKGBUILD

```diff
@@ -0,0 +1,57 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stringsifter
+pkgver=14.f4331a0
+pkgrel=1
+pkgdesc='Machine learning tool that automatically ranks strings based on their relevance for malware analysis.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='ihttps://github.com/fireeye/stringsifter'
+license=('Apache')
+depends=('python' 'python-virtualenv' 'python-numpy' 'python-scikit-learn' \
+  'python-joblib' 'python-pytest')
+makedepends=('git' 'python-setuptools')
+source=("git+https://github.com/fireeye/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+
+  python setup.py build
+}
+
+package() {
+  cd $pkgname
+
```

Comment:
## Setup.py

There is a setup.py file so why not just using `python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build` ?

## jointscript

Also if you use the jointscript method the `build()` method is not used.
You forgot to remove the `tests` folder.